tests/pdk/finfet/test_placement_grid.py

```diff
@@ -0,0 +1,29 @@
+import json
+try:
+    from .utils import get_test_id, build_example, run_example
+    from . import circuits
+except BaseException:
+    from utils import get_test_id, build_example, run_example
+    import circuits
+
+
+def test_place_on_grid():
+    name = f'ckt_{get_test_id()}'
+    netlist = circuits.tia(name)
+    setup = ""
+    constraints = []
+    example = build_example(name, netlist, setup, constraints)
+    _, run_dir = run_example(example, cleanup=False)
+
+    with (run_dir / '3_pnr' / 'Results' / f'{name.upper()}_0.placement_verilog.json').open('rt') as fp:
+        d = json.load(fp)
+        found = False
+        for m in d['modules']:
+            for i in m['instances']:
+                if i['abstract_template_name'] == 'TFR_PRIM_L_1E06_W_1E06':
+                    t = i['transformation']
+                    assert t['sX'] == 1
+                    assert t['sY'] == 1
+                    assert t['oX'] % 12600 == 6300, f'primitive is not placed on grid {t}'
```

Comment:
PnRDB uses different unit from json unit. It will need some transformation. 
In PnR, it is `t['oX'] % 1080 == 216`, `t['oY'] % 2520 == 1260`

Reply:
12600/ScaleFactor*2=2520, where ScaleFactor=10

Reply:
After I changed this line, the pytest is correct.

Reply:
The test is updated with scaled_placement_verilog.json